sphinx_rtd_theme/layout.html

```diff
@@ -6,10 +6,11 @@
 {%- else %}
   {%- set titlesuffix = "" %}
 {%- endif %}
+{%- set lang_attr = 'en' if language == None else (language | replace('_', '-')) %}
```

Comment:
I've checked the output of

    sphinx-build demo_docs/source demo_docs/build
    sphinx-build demo_docs/source demo_docs/build -A language=pt_BR

In the first case language is None so `en` is used. I can update it to handle things like empty string if we're expecting people to pass `-A language=`. I was taking a cue from the official basic theme which just checks None-ness (though it uses `is none` which equivalent but looks like the more "correct" way to do it, so happy to update the syntax there).

https://github.com/sphinx-doc/sphinx/blob/master/sphinx/themes/basic/layout.html#L120